mlocal/frags/go_common_opts.mk

```diff
@@ -1,11 +1,4 @@
 # go tool default build options
-GO_TAGS := -tags "containers_image_openpgp apparmor selinux"
```

Comment:
By removing `-tags` from this initial GO_TAGS definition we can add to the list as we build up the correct set through the various frags included after this.